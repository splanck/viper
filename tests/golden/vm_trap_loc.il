il 0.1
func @main() -> i64 {
entry:
  .loc 1 1 1
  trap
  ret 0
}
