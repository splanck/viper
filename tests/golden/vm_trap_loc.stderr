Trap @main#0 line 1: DomainError \(code=0\)
